core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -185,7 +197,7 @@ var TopicsAndSkillsDashboardPage = function() {
   };
 
   this.createSkillWithDescriptionAndExplanation = function(
-      description, reviewMaterial) {
+      description, reviewMaterial, closeSkillEditor) {
```

Comment:
ditto: Prefer nouns to verbs for variables

Reply:
Done